tensor_api.ipynb

```diff
@@ -0,0 +1,563 @@
+{
```

Comment:
<ol><li>The comments are wrong. The cecond one is host to device. The thrid one is device to another device.</li><li>Print <code>o3d.core.Device(xxx)</code> to show that it creates a device object.</li><li>(If a two-GPU machine is needed, please use the google cloud instance.)</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/intel-isl/Open3D/pull/2067/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>